### PR TITLE
[cli] Remove `qs` dependency

### PR DESCRIPTION
**Pull request by styfle**
There is no need for `qs` because the query string for `teamId` is already handled here:

https://github.com/vercel/vercel/blob/5b88f673f89c44b9e91e44247e29be6bed43805a/packages/cli/src/util/client.ts#L99

### CODE REVIEW

File: packages/cli/src/commands/git/index.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export default async function main(client: Client) {
   }
 
   const { org, project } = linkedProject;
+  client.config.currentTeam = org.type === 'team' ? org.id : undefined;
```
Comment (TooTallNate) [inline packages/cli/src/commands/git/index.ts:99]: Is this change related?

Comment (styfle) [inline packages/cli/src/commands/git/index.ts:99]: @TooTallNate Yes, this is used for all client fetch calls here

https://github.com/vercel/vercel/blob/5b88f673f89c44b9e91e44247e29be6bed43805a/packages/cli/src/util/client.ts#L99

**Review by TooTallNate — APPROVED** [review]
